src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -8,11 +8,11 @@
 /**
  * Lists all students in the address book to the user.
```

Comment:
Not sure if should change to include sessions as well? so 'lists all students and sessions'

Reply:
Haha nice catch! Changed that in the latest commit liao!